cypress/integration/version/version_route.ts

```diff
@@ -87,7 +87,17 @@ describe("Version route", () => {
     });
 
     it("Navigates to task tab and applies filters when clicking on grouped task status badge", () => {
+      // click on a different tab first, so that we aren't on the task tab initially
+      cy.dataCy("changes-tab").first().click();
+      cy.dataCy("task-tab")
+        .invoke("attr", "aria-selected")
```

Comment:
I'm not familiar with `invoke()`, but the docs suggest that it should be used to run a function. Perhaps the "should have" syntax used [here](https://github.com/evergreen-ci/spruce/blob/main/cypress/integration/version/unscheduled_patch/configure_patch.ts#L40-L44) could get the job done instead?